src/poprox_storage/repositories/data_stores/db.py

```diff
@@ -16,10 +18,67 @@
 logger.setLevel(logging.DEBUG)
 
 
+DbRepositories: type[BaseModel] = create_model(
+    "DbRepositories",
+    conn=(Connection, None),
+    __config__=ConfigDict(arbitrary_types_allowed=True),
+)
```

Comment:
This dynamically constructs a Pydantic model with a field for the database connection, which provides a default value here in case we don't detect any sub-classes of `DatabaseRepository` and establishes `DbRepositories` as a thing before we get to the decorator below